jumpscale/packages/marketplace/frontend/data.js

```diff
@@ -1,102 +1,168 @@
 const SECTIONS = {
-  "DECENTRALIZED WE": {
-    titleToolTip: null,
-    apps: {
-      peertube: {
-        name: "Peertube",
-        type: "peertube",
-        image: "./assets/peertube.png",
-        disable: false,
-        helpLink: "https://now10.threefold.io/docs/video-sharing/",
-        description: "Peertube is an open-source video platform that uses peer-to-peer technologies to reduce load on individual servers when viewing videos."
-      },
-      website: {
-        name: "Website",
-        type: "website",
-        image: "./assets/web.png",
-        disable: false,
-        helpLink:"https://now10.threefold.io/docs/website-publisher/",
-        description: "Website is a P2P alternative to centralized cloud-hosted websites. Host your own website with access via a public web address."
-      },
-      wiki: {
-        name: "Wiki",
-        type: "wiki",
-        image: "./assets/doc-flat.svg",
-        disable: false,
-        helpLink: "https://now10.threefold.io/docs/wiki-publisher/",
-        description: "Wiki is a versatile online encyclopedia builder, accessible via a public web address."
-      },
-      blog: {
-        name: "Blog",
-        type: "blog",
-        image: "./assets/blog.png",
-        disable: false,
-        helpLink: "https://now10.threefold.io/docs/blog-publisher/",
-        description: "Blog is a P2P alternative to centralized blogging platforms like Tumblr or Blogspot."
-      },
+    "All Solutions": {
```

Comment:
each section should be defined in a separate const and all solutions should use spread operator on all sections known